File: TesteAndroidv2/app/src/main/java/com/everis/TesteAndroidv2/Login/Model/LoginData.java
package com.everis.TesteAndroidv2.Login.Model;

import com.everis.TesteAndroidv2.Base.Model.Error;

public class LoginData {
    private UserAccount userAccount;
    private Error error;

    public UserAccount getUserAccount() {
        return userAccount;
    }

    public Error getError() {
        return error;
    }
}

File: app/src/main/java/pt/felipegouveia/bankapp/data/login/model/LoginData.kt
package pt.felipegouveia.bankapp.data.login.model

import com.google.gson.annotations.SerializedName
import pt.felipegouveia.bankapp.data.common.Error

data class LoginData(
    @SerializedName("userAccount")
    val userAccount: UserAccountData?,
    @SerializedName("error")
    val error: Error?
)